web3.js/src/transaction.ts

```diff
@@ -542,6 +543,21 @@ export class Transaction {
     this.signatures[index].signature = Buffer.from(signature);
   }
 
+  async getEstimatedFee(connection: Connection): Promise<RpcResponseAndContext<number>> {
```

Comment:
The `Transaction` class methods don't return RPC response contexts, we should unwrap the value. I think only the raw `Connection` methods do this.

Reply:
@jacobcreech suggested adding the return type, which in this case is `Promise<RpcResponseAndContext<number>>` since it's calling a `Connection` method. Do you have any suggestions on how I should handle this?

Reply:
My view is that none of the `Transaction` class API returns this context. If someone wants it, they can use `Connection` directly, since this is just a thin wrapper around it anyway.

Reply:
nit: could we move this function to right after `compileMessage` since it calls that? Right now it's sandwiched between some signing methods.

Reply:
Got it. Makes sense. And yes, I will move it. Thanks for the feedback!

Reply:
If context isn't necessary for methods of the `Transaction` class, would you say returning only the `value` works? That way I can make the return type `number`.

Reply:
Yep, exactly!

Reply:
Great! Let me know if there is anything else you would like me to change. I know I forgot to add a comment to describe the purpose of the method, so I'm going to do that now.

Reply:
Looks good!